sys/net/gnrc/routing/simple_subnets/simple_subnets.c

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
And

`sys/net/gnrc/routing/simple_subnets/simple_subnets.c` ⇒ `sys/net/gnrc/routing/ipv6_simple_subnets/gnrc_ipv6_simple_subnets.c`